src/main/java/org/sagebionetworks/web/server/servlet/filter/CrawlFilter.java

```diff
@@ -116,25 +114,22 @@ public String get() {
 		};
 	}
 
-	/**
-	 * Injected with Gin
-	 */
-	private SynapseClientImpl synapseClient;
-	private DiscussionForumClientImpl discussionForumClient;
-	JSONObjectAdapter jsonObjectAdapter;
-
-	@Override
-	public void destroy() {
-		sc = null;
+	public void init(SynapseClientImpl synapseClient, DiscussionForumClientImpl discussionForumClient) {
```

Comment:
Way to set the SynapseClient and DiscussionForumClient implementations (to allow us to mock in the test)